my_enumerable.rb

```diff
@@ -0,0 +1,18 @@
+# Module MyEnumerable
+module MyEnumeralbe
+  def all?
+    each { |item| return false unless yield item }
+    true
+  end
+
+  def any?
+    each { |item| return true if yield item }
+    false
+  end
+
+  def filter
+    results = []
+    each { |item| results << element if yield item }
```

Comment:
- [ ] Great job so far. However, for the filter method. I get this error:  ```undefined local variable or method `element' for #<MyList:0x000056543ac4dfa8 @list=[1, 2, 3, 4]> (NameError)```. I will suggest you change `element` to `item` so your  method can work.